### PR TITLE
add video playlist links

### PR DESCRIPTION
**Pull request by bahmutov**
Added links of Cypress short videos playlists

People love Bushra's videos https://twitter.com/imBushraAlam/status/1177557719617654784

### CODE REVIEW

File: source/examples/examples/tutorials.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ First we'll test our "Login Page", refactor, and then create a {% url "Custom Co
 | -- | -- | -- | -- |
 | {% url "Organizing Tests, Logging In, Controlling State" https://www.youtube.com/watch?v=5XQOK0v_YRE %} | {% fa fa-video-camera %} 27:21 | <time datetime="2018-02-22T16:00:00.000Z">02-22-2018</time> | 2.1.0 |
 
+# Video playlists
```
Comment (jennifer-shehane) [inline source/examples/examples/tutorials.md:18]: This seems out of place - the intro section says "In this tuturial we will" indicating this is a single tutorial to follow and then this section comes up in between this and the start of the tutorial.

File: source/examples/examples/tutorials.md [hunk 2]
```diff
@@ -5,16 +5,25 @@ containerClass: examples
 
 # Best Practices
 
-In this tutorial, we'll focus on writing and organizing tests for a real application.
+This video by the creator of Cypress Brian Mann shows what we consider the best practices for writing tests for a realistic application.
```
Comment (jennifer-shehane) [inline source/examples/examples/tutorials.md:8]: This isn't really a tutorial though. It's a talk. I would remove this section, we have this linked in other places. This page is for tutorials to walk someone through testing something from start to finish. 

The other section looks good now though - having the list above where the other tutorial starts.

Comment (bahmutov) [reply source/examples/examples/tutorials.md:8]: I would not remove it just yet, this is an extremely popular video, I would punt on the removal

Comment (jennifer-shehane) [reply source/examples/examples/tutorials.md:8]: Just because it's popular doesn't mean it belongs in this section of our docs.

Comment (bahmutov) [reply source/examples/examples/tutorials.md:8]: :) probably, but maybe we can handle it separately from adding more video links

Comment (bahmutov) [reply source/examples/examples/tutorials.md:8]: I have opened a separate issue to move it https://github.com/cypress-io/cypress-documentation/issues/2912

**Pull request merged**